Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,6 +37,8 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#define INVALID_HANDLE_VALUE -1
```

Comment:
Are you trying to redefine (or overwrite) a default INVALID_HANDLE_VALUE of an available library "handleapi.h"?